DetectorTomography.py

```diff
@@ -44,137 +44,46 @@ def gtc_matrix_product_counting_function(arguments: list):
     return result
 
 
-class DetectorTomographyFitter:
-    """
-        This class is meant to resemble qiskit's state tomography and process tomography fitters and to calculate the
-        maximum likelihood povm estimator describing a detector basing on QDT job results and used probe states.
-    """
+class QDTCalibrationSetup:
 
-    def __init__(self, algorithm_convergence_threshold=1e-6):
-        self.algorithmConvergenceThreshold = algorithm_convergence_threshold
+    def __init__(self, qubits_number: int, probe_kets: List[np.array], frequencies_array: np.ndarray ):
+        self.qubits_number = qubits_number
+        self.probe_kets = probe_kets
+        self.probe_states = self.__get_probe_states(qubits_number, probe_kets)
+        self.frequencies_array = frequencies_array
 
-    def get_maximum_likelihood_povm_estimator(self, results_list: List[Result],
-                                              probe_kets: List[np.array], qiskit_register_convention=True) \
-            -> List[np.ndarray]:
+    @classmethod
+    def from_qiskit_results(cls, results_list: List[Result], probe_kets: List[np.array]):
```

Comment:
We need to specify what is "cls". Honestly, I don't get it from the context. :-D

Reply:
Cls is not an argument. It's a Python keyword for "class". Here's one of stack overflow discussion:
https://stackoverflow.com/questions/4613000/what-is-the-cls-variable-used-for-in-python-classes.